README.md

```diff
@@ -60,8 +59,6 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ OPTIONAL] Super sorry to inline in your readme since I was not able to inline in a particular file :pray: ,  Your code looks clean and well-organized. The use of the Routes and Route components from react-router-dom is a good practice for handling the navigation between pages. The NavBar component is a nice touch as well and adds to the overall user experience. :+1:  , But you still need to improve in the `Calculator.js` component. It currently doesn't have any child components, which can make the code harder to maintain and organize in the long run. Adding at least one child component to the `Calculator.js` component would help to make the code more modular and easier to work with in the future. 
![Screenshot from 2023-04-24 16-33-51](https://user-images.githubusercontent.com/88894525/234028859-2d60a1d7-4b6f-4bc0-a970-409bb9aea373.png)

_note: To improve your code, I would suggest modifying the App.js file to import the Calculator component directly and render it as the main component. This will simplify the code and make it easier to read and maintain._